### PR TITLE
Fixed crawler tests and added multiple db tests configuration

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: germanium/test_cases/default.py [hunk 1]
```diff
@@ -3,9 +3,13 @@
 from django.conf import settings
 from django.test.testcases import TestCase, SimpleTestCase
 
+from germanium.config import MILTI_DB_TESTS
+
 
 class GermaniumSimpleTestCaseMixin:
 
+    multi_db = MILTI_DB_TESTS
```
Comment (matllubos) [inline germanium/test_cases/default.py]: this parameter is removed in django 3.0 and is undocumented :(

File: germanium/crawler.py [hunk 2]
```diff
@@ -152,8 +152,7 @@ def _call_request(self, url_with_referer):
                         not any(excluded_urls_regex.match(parsed_url)
                                 for excluded_urls_regex in self.excluded_urls_regexs)):
                     self.urls.add(URLWithReferer(parsed_url, url))
-            e = None
+            self._post_response(url, referer, resp)
         except Exception as e:
             LOG.exception('%s had unhandled exception: %s', url, e)
-
-        self._post_response(url, referer, resp, e)
+            self._post_response(url, referer, resp, e)
```
Comment (matllubos) [inline germanium/crawler.py]: @Formulka finally

Comment (Formulka) [reply germanium/crawler.py]: holy shit :bowdown:

File: germanium/test_cases/default.py (hunk 1 above)
Comment (Formulka) [reply germanium/test_cases/default.py]: imho it's only deprecated and there is a replacement 🤷‍♂

File: germanium/config.py [hunk 3]
```diff
@@ -33,3 +33,5 @@
 WINDOW_SIZE = (1124, 850)
 
 TURN_OFF_MAX_DIFF = getattr(settings, 'TURN_OFF_MAX_DIFF', True)
+
+MILTI_DB_TESTS = getattr(settings, 'GERMANIUM_MILTI_DB_TESTS', False)
```
Comment (rubickcz) [inline germanium/config.py]: Maybe you meant "MULTI"?

File: germanium/test_cases/default.py (hunk 1 above)
Comment (matllubos) [reply germanium/test_cases/default.py]: django 3.0 it just remove it means that

* default value for django<3.0 is false
* default and only value for django >= 3.0 is true

File: germanium/config.py (hunk 3 above)
Comment (matllubos) [reply germanium/config.py]: yes thanks

**Pull request merged**